README.md

```diff
@@ -1,7 +1,7 @@
 Glide
 =====
 
-[![Build Status](https://travis-ci.org/bumptech/glide.svg?branch=master)](https://travis-ci.org/bumptech/glide)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg) [![Build Status](https://travis-ci.org/bumptech/glide.svg?branch=master)](https://travis-ci.org/bumptech/glide)
```

Comment:
Yes, but when a user clicks on it, it opens https://maven-badges.herokuapp.com/maven-central/com.github.bumptech.glide/glide/badge.svg which is pretty useless. It should open something like this: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.bumptech.glide%22%20AND%20a%3A%22glide%22 not sure what other repos do.